cloud.foundry.cli/src/main/java/cloud/foundry/cli/crosscutting/annotations/OnlySupportedChanceTypes.java

```diff
@@ -0,0 +1,38 @@
+//package cloud.foundry.cli.crosscutting.annotations;
```

Comment:
Why is this class commented out?
New line missing at the end

Reply:
File has been removed.